crates/services/consensus_module/poa/src/service_test.rs

```diff
@@ -168,18 +177,26 @@ impl TestContextBuilder {
 
         let predefined_blocks = HashMap::new().into();
 
+        let time = match self.start_time {
+            Some(start_time) => TestTime::new(start_time),
+            None => TestTime::at_unix_epoch(),
+        };
```

Comment:
if you give TestTime a Default implementation that returns `TestTime::at_unix_epoch()`, you can reduce this to 

```rust
        let time = self.start_time.map(TestTime::new).unwrap_or_default();
```

Reply:
Good point, will do.

Reply:
Done in https://github.com/FuelLabs/fuel-core/pull/2145/commits/288d9031ec8207c1375a8882ca733c41fc5c9486